src/mudlet.cpp

```diff
@@ -2067,9 +2067,12 @@ void mudlet::slot_show_help_dialog_download()
 
 void mudlet::slot_show_about_dialog()
 {
-    auto pDlg = new dlgAboutDialog( this );
-    pDlg->raise();
-    pDlg->show();
+    if( ! mpAboutDlg ) {
+        mpAboutDlg = new dlgAboutDialog( this );
```

Comment:
Once we have got the `mpAboutDlg` pointer aimed at the dialogue we created we then have to make sure it gets cleared when the dialogue goes away again when the user clicks on the "close" button.  I think the way to do this is to change the `mpAboutDlg` from being declared in the header as:

    dlgAboutDialog* mpAboutDlg;

to:

    QPointer<dlgAboutDialog> mpAboutDlg;

and then set a specific flag on the dialogue at this point in the code:

    mpAboutDlg->setAttribute( Qt::WA_DeleteOnClose, true );

{although the second argument defaults to true and can omitted - you will see this in two other places in the source code both in the `mudlet` class file as it happens... :grinning:}.  This should, if I understood it correctly, mean that when the "close" button is hit on the "about" dialogue the Qt Library core code will auto-magically arrange for it to be destroyed and because a `QPointer` is being used, that will automatically set itself to zero when the thing it _was_ pointing at gets destroyed - returning the state of affairs to that when the application was started... :nerd_face: